example_common_js_app_utils/reducers/app.js

```diff
@@ -0,0 +1,39 @@
+import ACTION_TYPES from '../actions/action_types';
```

Comment:
`app` seems to general for this, I think. Seems more like a `publicNames` reducer or?